clang/test/SemaCXX/constexpr-if.cpp

```diff
@@ -0,0 +1,10 @@
+// RUN: %clang_cc1 -std=c++26 -verify %s
```

Comment:
I think it would be better to verify that this is OK since C++20.

Reply:
@frederick-vs-ja I've updated the test